### PR TITLE
zh-translation:/docs/tasks/traffic-management/egress/egress-tls-origination/index.md

### PR DESCRIPTION
**Pull request by ExBs2724**
ref: https://github.com/servicemesher/istio-official-translation/issues/1325

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md [hunk 1]
```diff
@@ -1,64 +1,56 @@
...
-## Before you begin
+## 开始之前{#before-you-begin}
 
-* Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
+*   根据[安装指南](/zh/docs/setup/)中的说明部署 Istio。
 
-*   Start the [sleep]({{< github_tree >}}/samples/sleep) sample which will be used as a test source for external calls.
-
-    If you have enabled [automatic sidecar injection](/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection), deploy the `sleep` application:
+*   如果启用了 [Sidecar 的自动注入功能](/zh/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection)，运行：
```
Comment (ilylia) [inline content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md]: 缺了一句

File: content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md [hunk 2]
```diff
@@ -1,64 +1,56 @@
...
     $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@)
     {{< /text >}}
 
-    Note that any pod that you can `exec` and `curl` from will do for the procedures below.
+    实际上任何可以 `exec` 和 `curl` 的 Pod 都可以用来完成这一任务。
 
-*   Create a shell variable to hold the name of the source pod for sending requests to external services.
-    If you used the [sleep]({{< github_tree >}}/samples/sleep) sample, run:
+*   创建一个环境变量来保存用于将请求发送到外部服务的 pod 的名称。
+    在本任务中，我们将 `SOURCE_POD` 环境变量设置为已部署的 [sleep]({{< github_tree >}}/samples/sleep) pod：
```
Comment (ilylia) [inline content/zh/docs/tasks/traffic-management/egress/egress-tls-origination/index.md]: 与原文不一致

**Comment by ExBs2724** [comment]
@ilylia  这个改好了

**Pull request merged**